### PR TITLE
Add KUTTL job trigger for Manila

### PR DESCRIPTION
**Pull request by fmount**
We have patches in manila-operator [1] and the openstack related tools [2] that are supposed to bring support to kuttl tests.
This change adds the missing bits to trigger kuttl tests via prow jobs.

[1] https://github.com/openstack-k8s-operators/manila-operator/pull/98
[2] https://github.com/openstack-k8s-operators/install_yamls/pull/216

### CODE REVIEW

**Comment by fmount** [comment]
FYI @vhariria @gouthampacha we'll merge this once we have everything in place in the manila-operator repo.

**Review by viroel — CHANGES_REQUESTED** [review]
Thanks Francesco, see coments inline, since we recently migrate OCP cluster to 4.12.

File: ci-operator/config/openstack-k8s-operators/manila-operator/openstack-k8s-operators-manila-operator-main.yaml [hunk 1]
```diff
@@ -91,6 +103,17 @@ tests:
...
         CEPHFS
     workflow: openstack-k8s-operators-test-build-deploy-tempest
+- as: manila-operator-build-deploy-kuttl
+  cluster_claim:
+    architecture: amd64
+    cloud: vexxhost
+    owner: openstack-k8s-operators
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.10"
```
[the remark this answers is not in the document]
Comment (gouthampacha) [reply ci-operator/config/openstack-k8s-operators/manila-operator/openstack-k8s-operators-manila-operator-main.yaml:113]: ++

File: ci-operator/jobs/openstack-k8s-operators/manila-operator/openstack-k8s-operators-manila-operator-main-presubmits.yaml [hunk 2]
```diff
@@ -203,6 +203,68 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )manila-operator-build-deploy,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
[the remark this answers is not in the document]
Comment (gouthampacha) [reply ci-operator/jobs/openstack-k8s-operators/manila-operator/openstack-k8s-operators-manila-operator-main-presubmits.yaml:207]: the tests have merged now: https://github.com/openstack-k8s-operators/manila-operator/pull/98

**Comment by gouthampacha** [comment]
Duplicating this PR to rehearse the job with recent CI changes: https://github.com/openshift/release/pull/41744/

**Comment by fmount** [comment]
Closed in favor of: https://github.com/openshift/release/pull/41744

**Pull request closed without merging**